README.md

```diff
@@ -93,7 +93,7 @@ public class ExampleTestRunner extends AndroidJUnitRunner {
 To change settings on the device from your tests, just use the methods in the `TestButler` class. For example:
 
 ```java
-@Before
+@BeforeClass
```

Comment:
Yeah, it's somewhat debatable whether you should use `@Before` or `@BeforeClass`. It depends on your use case, IMO, and what you're trying to do. Either way this example is not right. It should either remove `static` and remove `Class` from the method names, or it should switch to `@BeforeClass` as this PR does.

I don't particularly care which one we pick, and this PR already uses `@BeforeClass`, so I'm fine merging it :)